gax-grpc/src/main/java/com/google/longrunning/OperationsClient.java

```diff
@@ -138,17 +142,28 @@ protected OperationsClient(OperationsSettings settings) throws IOException {
     this.executor = channelAndExecutor.getExecutor();
     this.channel = channelAndExecutor.getChannel();
 
+    CredentialsProvider credentialsProvider = settings.getCredentialsProvider();
+    CallCredentials callCredentials = null;
+    if (credentialsProvider.getCredentials() != null) {
+      callCredentials = MoreCallCredentials.from(credentialsProvider.getCredentials());
+    }
+    ClientInitContext clientContext =
+        ClientInitContext.newBuilder()
+            .setExecutor(this.executor)
+            .setChannel(this.channel)
+            .setCallCredentials(callCredentials)
+            .build();
```

Comment:
It occurs to me that `ChannelAndExecutor` and `ClientInitContext` are redundant. I that `ClientSettings` should construct this object.